libyul/backends/wasm/TextTransform.cpp

```diff
@@ -41,14 +42,21 @@ string TextTransform::run(wasm::Module const& _module)
 	string ret = "(module\n";
 	for (auto const& sub: _module.subModules)
 		ret +=
-			"    ;; sub-module \"" +
+			"    ;; custom section for sub-module\n"
+			"    ;; (@custom \"" +
 			sub.first +
-			"\" will be encoded as custom section in binary here, but is skipped in text mode.\n";
+			"\" \"" +
+			// Transform text representation to hex string to avoid dealing with new lines.
+			toHex(asBytes(run(sub.second))) +
```

Comment:
Hmm, logically speaking, if we encode it as a hex string here, shouldn't it be the binary output instead?

Reply:
Implemented this in the last commit, I think that may be better.

Reply:
Yeah, I think having binary here makes more sense since the biggest use case for Yul subobjects seems to be storing the runtime bytecode and that wouldn't be deployable in text form, right?